pkg/confidence/confidence.go

```diff
@@ -83,10 +83,14 @@ func (e Confidence) PutContext(key string, value interface{}) {
 	e.contextMap[key] = value
 }
 
-func (e Confidence) Track(ctx context.Context, eventName string, message map[string]interface{}) *sync.WaitGroup {
-	newMap := e.GetContext()
+func (e Confidence) Track(ctx context.Context, eventName string, data map[string]interface{}) *sync.WaitGroup {
```

Comment:
nit: eventName should be the first argument

Reply:
the first context is coming from go, it's not confidence context. i think it's pretty standard to have the go context as the first parameter. you still think we should move it?

Reply:
Ah you are right, no probably makes sense to keep it there then